test/boilerplate_suggestors/boilerplate_utilities_test.dart

```diff
@@ -12,59 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:convert';
-
 import 'package:analyzer/dart/analysis/utilities.dart';
 import 'package:analyzer/dart/ast/ast.dart';
+import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_constants.dart';
 import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
 import 'package:test/test.dart';
 
-const reportJson = r'''{
-  "exports": {
-    "lib/web_skin_dart.dart/ButtonProps": {
-      "type": "class",
-      "grammar": {
-        "name": "ButtonProps",
-        "meta": ["@Props()"]
-      }
-    },
-    "lib/web_skin_dart.dart/BarProps": {
-      "type": "class",
-      "grammar": {
-        "name": "BarProps",
-        "meta": ["@Props()"]
-      }
-    },
-    "lib/web_skin_dart.dart/BarState": {
-      "type": "class",
-      "grammar": {
-        "name": "BarState",
-        "meta": ["@State()"]
-      }
-    },
-    "lib/web_skin_dart.dart/BarPropsMixin": {
-      "type": "class",
-      "grammar": {
-        "name": "BarPropsMixin",
-        "meta": ["@Props()"]
-      }
-    },
-    "lib/web_skin_dart.dart/BarStateMixin": {
-      "type": "class",
-      "grammar": {
-        "name": "BarStateMixin",
-        "meta": ["@State()"]
-      }
-    },
-    "lib/another_file.dart/ButtonProps": {
-      "type": "class",
-      "grammar": {
-        "name": "ButtonProps",
-        "meta": ["@Props()"]
-      }
-    }
-  }
-}''';
+String exportLocationsComment(List<String> locations) =>
```

Comment:
I'm not sure this is the best home for this utility, but it also doesn't really have a clear home haha. I'd probably either it make a new file in test/boilerplate_suggestors called "utils" or add it to the general test utils, since this file is more for testing the utilities from the boilerplate codemods and not so much the home of testing utils.

Reply:
Yeah I didn't know where to put it either haha. I moved it to the general test utils file, but if anyone thinks it should be in its own file or somewhere else let me know.